src/akkudoktoreos/class_akku.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         self.min_soc_wh = (self.min_soc_prozent / 100) * self.kapazitaet_wh
         self.max_soc_wh = (self.max_soc_prozent / 100) * self.kapazitaet_wh
 
-    def to_dict(self):
+    def return_dict(self):
```

Comment:
We should stick to methods names used in Python std libs (`to_dict`).

Reply:
yes please, return is uncommon

Reply:
removed